### ISSUE TITLE
Keyboard shortcuts for tabs don't match order of tabs

### ISSUE THREAD

**Issue by sequelpro** [sequelpro/sequelpro#1541]
Issue imported from Google Code: https://code.google.com/p/sequel-pro/issues/detail?id=1541
Reporter:a...@betterment.com Date:2013-01-24 13:08:39 Status:Accepted

<b>What steps will reproduce the problem?</b>
1. press cmd + 4

What is the expected output? go to the triggers tab
What do you see instead? it goes to the table info tab

What version of Sequel Pro are you using? 1.0 RC1
What version of OS X are you using? 10.7
What version of MySQL are you connecting to on the server? 5.1

<b>Please provide any additional information below.</b>
I noticed that the Triggers tab has a keyboard shortcut of cmd + 6,
basically the ordering of shortcuts doesn't match the order of tab icons in the app.

**Comment by nacho4d** [comment]
Is somebody working on this? If not I can send a pull request :+1:

**Comment by nacho4d** [comment]
I just sent a pull request solving this issue with minimal changes.
https://github.com/sequelpro/sequelpro/pull/2208
Hope it gets merged :)

**Comment by chrisminett** [comment]
Changing existing keyboard shortcuts is not cool. I'm now forever switching to the wrong tab. Muscle-memory!

www.sequelpro.com/docs/Keyboard_Shortcuts

**Comment by nacho4d** [comment]
Sorry, I made the changes. I wasn't aware of such page. 

We need to do one of the following:

- Update the shortcuts to match the docs and 
change the order of icons in the tab and order or menus like cmd+1,cmd+2, cmd+3 ... (I can do this in a pull request quickly)
- Update the docs to match current implementation (I don't think I can update this myself)

@chrisminett @sequelpro What do you think it is better?

**Comment by chrisminett** [comment]
I understand the reasoning, however I expect there will be more users out there which would be confused by the shortcuts changing. Perhaps follow the route suggested in https://github.com/sequelpro/sequelpro/issues/1541#issuecomment-28119763 and re-order the default toolbar layout to match the original keyboard shortcuts?

**Comment by dmoagx** [comment]
I'd like to get that sorted before the final 1.1 release.

Visually speaking I prefer the "Query" tab to be the right-most, as it is different from all the others.
This should of course match with the order in the "View" menu and the order of the keyboard shortcuts.

Historically speaking, the Query tab was added before Triggers and Relations, that is why it had a different keyboard shortcut.

Now, I don't really mind the keyboard shortcuts changing. Destroying established workflows is a totally normal thing under OS X, as regularly practiced by Apple. :imp: 

I think the change is fine and we just need to update the docs.
You still should be able to add a custom override via System Prefs > Keyboard > Shortcuts > App Shortcuts.
